src/loggers.jl

```diff
@@ -460,6 +460,12 @@ let
                         end
                         throw(exc)
                     end
+
+                    function $level(logger::Logger, msg::AbstractString, exc::Exception)
+                        e = ErrorException(msg * sprint(showerror, exc))
+                        log(logger, $key, sprint(showerror, e))
```

Comment:
Could we eliminate a call to `showerror`?
```julia
msg *= sprint(showerror, exc)
e = ErrorException(msg)
log(logger, $key, msg)
throw(e)
```

Reply:
Sure that would work